### PR TITLE
CSI: Recover the workloads with stale COS Volume

### PR DESCRIPTION
**Pull request by ashimagarg27**
Test Results: https://github.com/ambiknai/csi-plugin-operator/pull/1

### CODE REVIEW

File: api/v1alpha1/fixstalevolume_types.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2023.
```
Comment (ambiknai) [inline api/v1alpha1/fixstalevolume_types.go:2]: Can we make it 2024

File: controllers/fixstalevolume_controller.go [hunk 2]
```diff
@@ -0,0 +1,320 @@
+/*
+Copyright 2023.
```
Comment (ambiknai) [inline controllers/fixstalevolume_controller.go:2]: same here

**Review by ambiknai — CHANGES_REQUESTED** [review]
lgtm. Pls fix copyright

**Review by ambiknai — APPROVED** [review]

**Review by nkkashyap — APPROVED** [review]
lgtm

**Comment by minbaev** [comment]
lgtm

**Pull request merged**